components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -936,12 +936,13 @@ private String buildKeySetString(String[] keyArray){
      * @return a List of APIVersionUsageDTO objects, possibly empty
      * @throws org.wso2.carbon.apimgt.usage.client.exception.APIMgtUsageQueryServiceClientException on error
      */
-    public List<APIVersionUsageDTO> getUsageByAPIVersions(String providerName,
-                                                          String apiName) throws APIMgtUsageQueryServiceClientException {
+    public List<APIVersionUsageDTO> getUsageByAPIVersions(String providerName, String apiName)
+            throws APIMgtUsageQueryServiceClientException {
 
-        OMElement omElement = this.queryBetweenTwoDaysForAPIUsageByVersion(
-                APIUsageStatisticsClientConstants.API_VERSION_USAGE_SUMMARY, null, null, apiName);
-        Collection<APIUsage> usageData = getUsageData(omElement);
+        List<APIUsage> usageData = this
+                .queryBetweenTwoDaysForAPIUsageByVersion(APIUsageStatisticsClientConstants.API_VERSION_USAGE_SUMMARY,
+                        null, null, apiName);
+//        Collection<APIUsage> usageData = getUsageData(omElement);
```

Comment:
content is replace in commit: https://github.com/wso2/carbon-apimgt/commit/078e20af796c5e61f8f9696b995e05cb82ff8953#diff-bb18fa9c4e9abc621195a0ecc75bef6d